html-css/lesson1.md

```diff
@@ -158,7 +158,12 @@ You can also set a transparent border, so that it takes up the space without sho
   - Proper use of the semantic HTML tags we discussed.
   - Research the `:first-child` pseudo-class and use it to style the first article so that it stands out from the others.
 
-2. **Optional** (Est. 1 hour) Read about [advanced CSS selectors](http://learn.shayhowe.com/advanced-html-css/complex-selectors/) and then practice by playing this [CSS selector game](https://flukeout.github.io/). It gets hard at the end, but try your best!
+2. (Est. 5 hours) FreeCodeCamp beginner lessons
```

Comment:
Thanks, good to know. Is that the only challenge they've done, then? Can I refer to any other challenges in the future? (like the Bootstrap/responsive one for the second lesson)